src/dataframe/dataframe.jl

```diff
@@ -1594,6 +1594,11 @@ type (except `AbstractArray`). In the latter case the passed value is treated
 as having length one for expansion. As a particular rule values stored in a `Ref`
 or a `0`-dimensional `AbstractArray` are unwrapped and treated as having length one.
 
+# See also
+                                                                                                    
+[`crossjoin`](@ref) can be used to get the cartesian product
```

Comment:
I think it's more common to just include a paragraph (that's what the Julia manual recommends doing).
```suggestion
See also: [`crossjoin`](@ref) can be used to get the cartesian product
```